docs/Deploying-Pega-on-openshift.md

```diff
@@ -0,0 +1,358 @@
+
+- pega/pega - Use this chart to set customization parameters for your deployment. You will modify this chart later in the deployment tasks.
+
+- pega/backingservices - Use this chart to set customization parameters for the Pega-provided Search and Reporting Service (SRS) your deployment. You will modify this chart later in the deployment tasks.
+
+To customize these files, you must download them from the source github repository to your local system, edit them with a text editor, and then save them to your local system using the same filename.
+
+1. To add the Pega repository to your Helm installation, enter:
+
+    `$ helm repo add pega https://pegasystems.github.io/pega-helm-charts`
+
+2. To verify the new repository, you can search it by entering:
+
+```
+  $ helm search repo pega
+  NAME                  CHART VERSION   APP VERSION     DESCRIPTION
+  pega/pega             1.6.1                           Helm chart to configure required installation and deployment configuration settings in your environment for your deployment.
+  pega/addons           1.6.1           1.0             Helm chart to configure required supporting services and tools in your environment for your deployment.
+  pega/backingservices  1.6.1                           Helm Chart to provision the latest Search and Reporting Service (SRS) for your Pega Infinity deployment
+```
+
+The addons charts is not required for OpenShift deployments of Pega Platform. Use of the backingservices chart is optional, but recommended for Pega Infinity 8.6 and later.
+
+#### Updating the backingservices.yaml Helm chart values for the SRS (Supported when installing or upgrading to Pega Infinity 8.6 and later)
+
+To configure the parameters in the backingservices.yaml file, download the file in the charts/backingservices folder, edit it with a text editor, and then save it to your local system using the same filename.
+
+Configure the following parameters so the backingservices.yaml Helm chart matches your Elastic resources in these areas:
+
+- Your docker registry credentials.
+
+- Your deployment name with which the deployment appends the names of your SRS pods.
+
+- Your Pega-provided SRS Docker image in the repository to which you pushed the image.
+
+- Your choice to enable either:
+
+  - An ElasticSearch service that is defined at [Elasticsearch Helm Chart](https://github.com/elastic/helm-charts/tree/master/elasticsearch) which the SRS provides to your deployment.
+  - Your Elasticsearch cluster for which you specify the required authentication details.
+
+- And additional, supported environmental variables for Elasticsearch basic authentication.
+
+- Three additional parameters that elasticSearch recommends for OpenShift deployments.
```

Comment:
If we are adding this statement, could you please provide the link here to https://github.com/elastic/helm-charts/blob/master/elasticsearch/examples/openshift/values.yaml 
Though we might need to update it with exact openshift specific settings later including setting xpack.security.transport.ssl.enabled: false

Reply:
I have now added these details to the runbook